scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java

```diff
@@ -349,6 +351,19 @@ public boolean kill() {
 
         listener.schedulerStateUpdated(SchedulerEvent.KILLED);
 
+        // To properly exit the java scheduling process
+        new Thread(new Runnable() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)